tests/filters/test_capfirst.py

```diff
@@ -0,0 +1,40 @@
+"""
+Adapted from
+https://github.com/django/django/blob/5.1/tests/template_tests/filter_tests/test_capfirst.py
+"""
+
+import pytest
+from django.utils.safestring import mark_safe
+
+
+@pytest.mark.fail(reason="autoescape not ready yet")
```

Comment:
I'm not sure when I'll get to it, but I plan to add `{% autoescape %}` support to #50 before merging it.